pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test: TestTenantLogic_zone_config failed

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19927118?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19927118?buildTab=artifacts#/) on master @ [af298151959414a065a2174187d5a012d07b268a](https://github.com/cockroachdb/cockroach/commits/af298151959414a065a2174187d5a012d07b268a):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config2524789449
    test_log_scope.go:76: use -show-logs to present logs inline
[16:29:47] setting distsql_workmem='62135B';
[16:29:47] setting distsql_workmem='62135B';
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[16:29:56] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30868/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[16:29:56] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config2524789449
--- FAIL: TestTenantLogic_zone_config (11.23s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[16:29:55] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30868/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 59 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30868/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            bar  03:25:45
            baz  04:00:00
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30868/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.24s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19927822?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19927822?buildTab=artifacts#/) on master @ [8138e3f859a140ced81a5c601b2b2d248e853ceb](https://github.com/cockroachdb/cockroach/commits/8138e3f859a140ced81a5c601b2b2d248e853ceb):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config647313031
    test_log_scope.go:76: use -show-logs to present logs inline
[17:52:26] setting distsql_workmem='17763B';
[17:52:26] setting distsql_workmem='17763B';
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[17:52:35] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31448/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[17:52:36] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config647313031
--- FAIL: TestTenantLogic_zone_config (11.11s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[17:52:35] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31448/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 59 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31448/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            bar  03:25:45
            baz  04:00:00
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31448/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.32s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19930505?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19930505?buildTab=artifacts#/) on master @ [c5954702f1e0a46b20595cbcc7f6a66380f2e6c1](https://github.com/cockroachdb/cockroach/commits/c5954702f1e0a46b20595cbcc7f6a66380f2e6c1):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config3040866440
    test_log_scope.go:76: use -show-logs to present logs inline
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[22:20:23] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31323/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[22:20:24] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config3040866440
--- FAIL: TestTenantLogic_zone_config (11.18s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[22:20:20] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31323/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 58 statements
[22:20:23] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31323/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 59 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31323/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            bar  03:25:45
            baz  04:00:00
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31323/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.32s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19931148?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19931148?buildTab=artifacts#/) on master @ [f48b3438a296aad0ff6e2ce88875c6021dfea872](https://github.com/cockroachdb/cockroach/commits/f48b3438a296aad0ff6e2ce88875c6021dfea872):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config2301656857
    test_log_scope.go:76: use -show-logs to present logs inline
[00:19:17] setting distsql_workmem='63981B';
[00:19:17] setting distsql_workmem='63981B';
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[00:19:26] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30874/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[00:19:26] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config2301656857
--- FAIL: TestTenantLogic_zone_config (11.16s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[00:19:23] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30874/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 57 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30874/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            bar  03:25:45
            baz  04:00:00
            
[00:19:26] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30874/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 60 statements
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30874/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.31s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_ScratchProjectPutTcExperimentsInHere_SqlLogicDeclarativeSchemaChangerCorpusBazel/19941927?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_ScratchProjectPutTcExperimentsInHere_SqlLogicDeclarativeSchemaChangerCorpusBazel/19941927?buildTab=artifacts#/) on master @ [28e1b226dfb3ceb38af3214d5b9e415d8e0c3c92](https://github.com/cockroachdb/cockroach/commits/28e1b226dfb3ceb38af3214d5b9e415d8e0c3c92):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/eede9103755ead07c3fa074ac2c1d0e5/logTestTenantLogic_zone_config838401779
    test_log_scope.go:76: use -show-logs to present logs inline
[05:51:14] setting distsql_workmem='50658B';
[05:51:14] setting distsql_workmem='50658B';
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[05:51:23] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4942/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[05:51:24] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/eede9103755ead07c3fa074ac2c1d0e5/logTestTenantLogic_zone_config838401779
--- FAIL: TestTenantLogic_zone_config (13.07s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[05:51:23] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4942/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 58 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4942/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            bar  03:25:45
            baz  04:00:00
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4942/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (6.85s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19942319?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19942319?buildTab=artifacts#/) on master @ [28e1b226dfb3ceb38af3214d5b9e415d8e0c3c92](https://github.com/cockroachdb/cockroach/commits/28e1b226dfb3ceb38af3214d5b9e415d8e0c3c92):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config2531338894
    test_log_scope.go:76: use -show-logs to present logs inline
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[06:07:22] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[06:07:23] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config2531338894
--- FAIL: TestTenantLogic_zone_config (10.73s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[06:07:22] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 59 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            baz  04:00:00
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.63s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19946476?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19946476?buildTab=artifacts#/) on master @ [ee5f7d28ff173b5b65d3293efae25b674183261e](https://github.com/cockroachdb/cockroach/commits/ee5f7d28ff173b5b65d3293efae25b674183261e):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config4080071748
    test_log_scope.go:76: use -show-logs to present logs inline
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[17:25:07] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/32021/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[17:25:08] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config4080071748
--- FAIL: TestTenantLogic_zone_config (11.00s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[17:25:07] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/32021/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 59 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/32021/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            baz  04:00:00
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/32021/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.34s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19947829?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19947829?buildTab=artifacts#/) on master @ [fea6fe50e0ebc842315a96ab4247df0f60784cf8](https://github.com/cockroachdb/cockroach/commits/fea6fe50e0ebc842315a96ab4247df0f60784cf8):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config2716607733
    test_log_scope.go:76: use -show-logs to present logs inline
[18:53:48] setting distsql_workmem='15206B';
[18:53:48] setting distsql_workmem='15206B';
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[18:53:57] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20553/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[18:53:57] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config2716607733
--- FAIL: TestTenantLogic_zone_config (11.11s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[18:53:56] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20553/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 59 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20553/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            bar  03:25:45
            baz  04:00:00
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20553/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.15s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19949456?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19949456?buildTab=artifacts#/) on master @ [8a1562c5cb076a6f61174b951544a8b9b3834923](https://github.com/cockroachdb/cockroach/commits/8a1562c5cb076a6f61174b951544a8b9b3834923):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config3645937750
    test_log_scope.go:76: use -show-logs to present logs inline
[21:28:43] setting distsql_workmem='95881B';
[21:28:43] setting distsql_workmem='95881B';
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[21:28:52] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31644/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[21:28:52] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config3645937750
--- FAIL: TestTenantLogic_zone_config (11.00s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[21:28:51] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31644/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 59 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31644/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            baz  04:00:00
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31644/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19950948?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19950948?buildTab=artifacts#/) on master @ [67cb06a1b702789469ccbccf8368d89790e7db03](https://github.com/cockroachdb/cockroach/commits/67cb06a1b702789469ccbccf8368d89790e7db03):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config3291719385
    test_log_scope.go:76: use -show-logs to present logs inline
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[01:23:06] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[01:23:07] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config3291719385
--- FAIL: TestTenantLogic_zone_config (10.46s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[01:23:05] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 59 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            baz  04:00:00
            vm   27:46:40
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (6.84s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19950963?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19950963?buildTab=artifacts#/) on master @ [67cb06a1b702789469ccbccf8368d89790e7db03](https://github.com/cockroachdb/cockroach/commits/67cb06a1b702789469ccbccf8368d89790e7db03):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config3706591081
    test_log_scope.go:76: use -show-logs to present logs inline
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[01:37:51] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[01:37:52] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config3706591081
--- FAIL: TestTenantLogic_zone_config (10.40s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[01:37:51] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 59 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            baz  04:00:00
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.03s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://tanzanite.cluster.engflow.com/invocations/default/13a29d76-59fe-485e-8df5-1aa99b8a48b9?testReportRun=10&testReportShard=38&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy8zbm9kZS10ZW5hbnQ6M25vZGUtdGVuYW50X3Rlc3Q=) on master @ [0d807294b480010fa253673775c4c15875db4bd4](https://github.com/cockroachdb/cockroach/commits/0d807294b480010fa253673775c4c15875db4bd4):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestTenantLogic_zone_config2872862828
    test_log_scope.go:76: use -show-logs to present logs inline
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[06:55:13] --- done: /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[06:55:14] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: outputs.zip/logTestTenantLogic_zone_config2872862828
--- FAIL: TestTenantLogic_zone_config (11.57s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[06:55:13] --- progress: /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 57 statements
    logic.go:3143: 
         
        /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            bar  03:25:45
            baz  04:00:00
            
    logic.go:2331: 
         /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.39s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=10</code>
 - <code>shard=38</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19953411?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19953411?buildTab=artifacts#/) on master @ [211a8ece7afb329b99030c9cd41c4e8e7cdb90ec](https://github.com/cockroachdb/cockroach/commits/211a8ece7afb329b99030c9cd41c4e8e7cdb90ec):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config1574003302
    test_log_scope.go:76: use -show-logs to present logs inline
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[10:20:00] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31624/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[10:20:01] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config1574003302
--- FAIL: TestTenantLogic_zone_config (11.06s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[10:20:00] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31624/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 59 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31624/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            baz  04:00:00
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31624/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.14s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19954804?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19954804?buildTab=artifacts#/) on master @ [975861fc5b8357323e7f6ab341c7685c00d138f4](https://github.com/cockroachdb/cockroach/commits/975861fc5b8357323e7f6ab341c7685c00d138f4):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config1776307180
    test_log_scope.go:76: use -show-logs to present logs inline
[13:41:10] setting distsql_workmem='67149B';
[13:41:10] setting distsql_workmem='67149B';
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[13:41:19] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31061/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[13:41:19] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config1776307180
--- FAIL: TestTenantLogic_zone_config (10.91s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[13:41:19] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31061/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 59 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31061/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            baz  04:00:00
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31061/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zone_config [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19955342?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19955342?buildTab=artifacts#/) on master @ [de926bac93941ba022be59c4e96a4cc30ff15f05](https://github.com/cockroachdb/cockroach/commits/de926bac93941ba022be59c4e96a4cc30ff15f05):


```
=== RUN   TestTenantLogic_zone_config
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/b8530b12366582f814734775ba1b4927/logTestTenantLogic_zone_config1109491439
    test_log_scope.go:76: use -show-logs to present logs inline
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[15:20:25] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31823/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config with config 3node-tenant: 60 tests, 2 failures
[15:20:26] --- total progress: 60 statements
--- total: 60 tests, 2 failures
    logic.go:4534: -- test log scope end --
--- FAIL: TestTenantLogic_zone_config (11.00s)
=== RUN   TestTenantLogic_zone_config/test_entity_validity
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports local-legacy-schema-changer (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
    logictestbase.go:551: statement/query skipped with reason: unsupported configuration 3node-tenant, statement/query only supports schema-locked-disabled (no issue given)
[15:20:25] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31823/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config: 59 statements
    logic.go:3143: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31823/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:423: SELECT name, ttl FROM crdb_internal.kv_dropped_relations ORDER BY name;
        expected:
            bar  03:25:45
            baz  04:00:00
            vm   27:46:40
            zc   27:46:40
            
        but found (query options: "") :
            bar  03:25:45
            baz  04:00:00
            
    logic.go:2331: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/31823/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config:431: too many errors encountered, skipping the rest of the input
--- FAIL: TestTenantLogic_zone_config/test_entity_validity (7.31s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zone_config.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/logictest/testdata/logic_test/zone_config
+++ b/pkg/sql/logictest/testdata/logic_test/zone_config
@@ -301,6 +301,10 @@ CREATE TABLE zc (
   b INT
 )
 
+# Prevent fast GC with data.
+statement ok
+INSERT INTO zc VALUES (1,2)
+
 statement ok
 ALTER TABLE zc CONFIGURE ZONE USING gc.ttlseconds = 100000
 
@@ -410,11 +414,19 @@ ALTER DATABASE foo CONFIGURE ZONE USING gc.ttlseconds = 12345
 statement ok
 CREATE TABLE foo.public.bar (x INT PRIMARY KEY)
 
+# Prevent fast GC with data.
+statement ok
+INSERT INTO foo.public.bar VALUES (1)
+
 statement ok
 DROP DATABASE foo CASCADE
 
 statement ok
 CREATE TABLE baz (x INT PRIMARY KEY)
+
+# Prevent fast GC with data.
+statement ok
+INSERT INTO baz VALUES (1)
 
 statement ok
 DROP TABLE baz

